### DISCUSSION TITLE
This line in code can be optimized

### DISCUSSION THREAD

**Discussion by classicrocker883** [mriscoc/Ender3V2S1#1112] [General]
https://github.com/mriscoc/Ender3V2S1/blob/4cc028a7c5ef277c36f2bc9fa619b6ec582105cb/Marlin/src/lcd/e3v2/proui/dwin.cpp#L699
```y
     _value = CEIL(MMS_SCALED(feedrate_mm_s));
```
of instead of `CEIL`, then `round` can be retained.

the reason being  
`#define MMS_SCALED(V) ((V) * 0.01f * feedrate_percentage)`  
is essentially the same as `(feedrate_mm_s * feedrate_percentage / 100)`  
however it is far better and faster for code to be _MULTIPLIED_ rather than use _DIVISION_

**Comment by mriscoc** [comment]
I remember that I keeped the division for intercept an old problem of lightly modified values when certains code is changed. Something similar to: https://github.com/MarlinFirmware/Marlin/issues/26196

That doesn't happen anymore, but I'm keeping it as a precaution.
